src/config/networks/local.ts

```diff
@@ -5,11 +5,18 @@ const baseConfig: EnvironmentSettings = {
   clientGatewayUrl: 'http://localhost:8001/v1',
   txServiceUrl: 'http://localhost:8000/api/v1',
   safeUrl: 'http://localhost:3000/app',
-  gasPriceOracle: {
-    url: 'https://ethgasstation.info/json/ethgasAPI.json',
-    gasParameter: 'average',
-    gweiFactor: '1e8',
-  },
+  gasPriceOracles: [
+    {
+      url: 'https://www.gasnow.org/api/v3/gas/price?utm_source=:gnosis',
```

Comment:
I suggest using `gnosis_safe` to make sure it's not confused with GP or something.

Also 
![image](https://user-images.githubusercontent.com/1380529/128138995-f1f2c5ef-1532-461b-af3b-f4cdd8312603.png)
Do we have any idea how frequently this gets called right now? Will be have this problem of getting rate limited? Or phrase differently: Would we notice that we get throttled via sentry or so?